### ISSUE TITLE
Feature Request: Add Action icons next to relevant feats that have them.

### ISSUE THREAD

**Issue by ThingsCouldGetDicey** [foundryvtt/pf2e#15841]
i would love a quick way to see the action economy next to all the feats on the feat page. currently the only way to find out the action economy is to click edit on the feat on the feat page. 

![chrome_we9Y1Zg4FS](https://github.com/user-attachments/assets/767a105d-8513-4329-9218-155086a59440)
it would be nice to have a column next to the feat description that has the action economy symbol.

**Comment by ammalagonc** [comment]
- [ ] Mighty Rage Automation (#15777)
- [ ] Shields of the Spirit + Security automation/effects (#15765)  
- [ ] Blessed Counterstrike RE/effect to add Weakness (#15761)
- [ ] Expand Aura - Track Aura/Effect duration tracking (#15760)
- [ ] Bravado Trait not showing in Skill Checks (#15754)
- [ ] Feature Request - Add a Highlight for Reach (#14797)
- [ ] Feature Request - make Shield Augmentations lie Shield attachments (#14694)
- [ ] Feature Request - Export Party Sheet Members into a Compendium (#14691)
- [ ] Feature Request - Add Attack Rolls as an option for Aid (#8148)
- [ ] Feature Request - Standardize Consumable usage (#8117)
- [ ] Add Damage Calculations to Wellspring Surge Table (#7780)
- [ ] Add currency to Actors from Earn Income Macro (#7328)
- [ ] Item Activations for Magic Items - Generate abilities/spells with set DCs (#7295)
- [ ] Automate Psychic -apply Stupefy after Unleash Psyche is done (#7208)
- [ ] Appy Temp HP to actors via system feature (#7177)
- [ ] Add Diplomacy/Attitude rules to GM Screen (#6340)
- [ ] Automate Soulforger Archetype  (waiting on afflictions(?)) (#5876)
- [ ] Overhaul visibility conditions (#5622)
- [ ] Walking Cauldron weight (make it dynamic) (#5064)
- [ ] Flanking getting applied when the actor is at a different height (#4875)
- [ ] Sync Torch to Game Clock - make torches timed effects (#4777)
- [ ] ABP with Lore Skill Proficiencies (#4620)
- [ ] Add setting to enable the display of text for conditions (#4495)
- [ ] Choose which skills to roll initiative from the Combat Tracker (#4267)
- [ ] Add sight-sharing mechanics to familiars (#3797)

**Comment by ThingsCouldGetDicey** [comment]
i dont know why im being singled out like this.
